spotbugs/src/main/java/edu/umd/cs/findbugs/TextUICommandLine.java

```diff
@@ -266,30 +270,40 @@ public boolean justPrintVersion() {
     private List<TextUIBugReporter> reporters = new ArrayList<>();
 
     /**
-     * Parse {@code optionExtraPart} and configure {@Link TextUIBugReporter} if it contains the
+     * Parse {@code optionExtraPart} create a path-associated {@Link TextUIBugReporter} if it contains the
      * output file path such as {@code ":withMessages=path/to/file.extension"} and {@code "=/absolute/path/to/file.extension"}.
+     * Finally configure the created BugReporter with the {@code optionExtraPart}'s configuration information.
      *
-     * @param reporter the reporter to set a {@link PrintStream} based on the given file path
      * @param optionExtraPart extra part of the specified commandline option
-     * @return Remaining part of {@code optionExtraPart}
+     * @param ctor            A supplier for an unconfigured {@Link TextUIBugReporter}.
+     * @param handleOptions   A function that can configure the created {@code BugReporter} instance.
+     * @param <T>             The implementation type of the {@Link TextUIBugReporter} to propagate type
+     *                        information between {@link ctor} and {@link handleOptions}.
+     * @return The fully configured reporter
```

Comment:
Can you please add a mention that this methods returns null in case the report was already produced?

Reply:
good shout, updated

Reply:
Thank you, LGTM!